Makefile

```diff
@@ -1,6 +1,9 @@
 altprodserver: migrate collectstatic ensurecrowdinclient downloadmessages compilemessages
 	cd contentcuration/ && gunicorn contentcuration.wsgi:application --timeout=4000 --error-logfile=/var/log/gunicorn-error.log --workers=3 --bind=0.0.0.0:8081 --pid=/tmp/contentcuration.pid --log-level=debug || sleep infinity
 
+demoserver: setup
```

Comment:
Rather than make a new make target, you can just do a `make setup altprodserver`, and it would run both!

Reply:
ah yes. thank you for pointing that out! didn't realize that while I was writing the rules.